media/js/fonts.js

```diff
@@ -0,0 +1,158 @@
+(function(win, doc, $) {
+    /*
+        Font Loading
+    */
+
+    if(window.waffle && window.waffle.flag_is_active('web_font_loader')) {
```

Comment:
On line 1 we have `win` which is an alias for window, so we can use that instead of `window` throughout this file.